services/softlayer_virtual_guest.go

```diff
@@ -15,7 +15,10 @@ import (
 )
 
 const (
-	EPHEMERAL_DISK_CATEGORY_CODE = "guest_disk1"
+	EPHEMERAL_DISK_CATEGORY_CODE      = "guest_disk1"
```

Comment:
Please provide links (as comments) to SL docs where these constant are defined... Thanks.